web/pandas/about/governance.md

```diff
@@ -197,103 +185,6 @@ communications and decisions are needed, the Core Team will do its best to
 summarize those to the Community after eliding personal/private/sensitive
 information that should not be posted to the public internet.
 
-### Subcommittees
-
-The Core Team can create subcommittees that provide leadership and guidance for
-specific aspects of the project. Like the Core Team as a whole, subcommittees
```

Comment:
In breaking this section up. we lose this sentence (and any mention of the word subcommittee). I guess that this sentence alone did not give much information but did show the possibility of forming subcommittees.

To avoid increasing the scope of this PR, i'm not suggesting any changes now as we can maybe update the governance again if we feel the need to create a subcommittee but just want to mention in case others have strong views.

Reply:
Good point. I added a section about it. I think we should have a "communications committee" too, which can better be discussed later. So, having a section about this seems worth having to me too.

I didn't realize before that I used `committee` instead of `subcommittee` in what I wrote. I'm personally happy with anything, committee, subcommittee, workgroup... I guess the idea in the current governance is that the core dev team is a committee and those are subcommittees of it. It felt more natural to call them committees, and also it doesn't seem technically correct to call them subcommittees, as for example the CoC committee is not made only of core devs. In any case, if people prefer to call them subcommittees, or something else, happy to update the PR.

Reply:
good point. committee is fine.